src/bin/driver.rs

```diff
@@ -264,7 +279,7 @@ mod app {
         local.dacs.1.start();
 
         // Spawn a settings update for default settings.
-        settings_update::spawn().unwrap();
+        settings_update::spawn(String::from("")).unwrap();
```

Comment:
This is where you'd use an `Option<String<64>>` don't encode that info in magic (empty) strings.